rfc/graphql-schema/additionalTypeDefs/byPage/invoices.ts

```diff
@@ -0,0 +1,143 @@
+export default /* GraphQL */ `
+  enum InvoiceStatus {
+    OPEN
+    PAID
+    OVERDUE
+    CANCELLED
+  }
+
+  type InvoiceStats {
+    openBalance: Float!
+    overdueBalance: Float!
+  }
+
+  input InvoiceDateRangeFilterInput {
+    from: DateTime!
```

Comment:
only from is mandatory? We don't want to offer "since forever"?

Reply:
From should be optional - we will want to allow the user to ask for all their invoices like Lee pointed out. Since this is entirely w/ b2b domain - we have the ability to implement it - let's go with that.

For orders we will probably need to ask the orders team what is feasible and have a discussion around it.

Reply:
made optional